common/shlibs

```diff
@@ -4325,7 +4326,7 @@ librz_type.so.0.7 rizin-0.7.2_1
 librz_util.so.0.7 rizin-0.7.2_1
 libaravis-0.8.so.0 libaravis-0.8.21_1
 libLimeSuite.so.23.11-1 LimeSuite-23.11.0_1
-libnvme.so.1 libnvme-1.0_1
+libnvme.so.1 libnvme-1.10_1
```

Comment:
this doesn't need to be changed unless the ABI of `libnvme` changed.

Reply:
Hi! 

Gimme a while and I'll test sth. The way I remember it was:
I compiled nvme-cli (w/out libnvme 1.10) and didn't receive any warning about the SHLIB in question. However, when launching nvme from CLI I was greeted with the message that it requires libnvme >=1.10, so I suppose ABI changed.

But to make sure, let me try to reproduce it.

Reply:
very well then. is this the only package that needs libnvme? if not, others need to be rebuilt

Reply:
I believe libblockdev is the only one missing. I'll add an appropriate commit. Thank you!